mcp/include/chrgctrl.h

```diff
@@ -45,7 +57,11 @@ along with mcp; if not, write to the Free Software Foundation, Inc.,
                                          ----------------------
                                          -                (6 B) */
 
-#define MAX_READ_REGS MAX_DATA_LENGTH/2  // each register is 2 bytes
+/**
+ * @brief maximum number of registers to read on modbus, each is 2 bytes
+ * 
+ */
+#define MAX_READ_REGS MAX_DATA_LENGTH/2
```

Comment:
For the same reasons as above, just to be safe, I'd put parentheses around `MAX_DATA_LENGTH/2` too.

Reply:
done